src/PerfView/PerfViewData.cs

```diff
@@ -7837,11 +7876,13 @@ private void AddResourcesAsChildren(StatusBar worker, DhPackage dhPackage, strin
         /// <summary>
         /// Adds child files from ETL resources in the DhPackage
         /// </summary>
-        private void AddEtlResourcesAsChildren(StatusBar worker, DhPackage dhPackage)
+        private void AddEtlResourcesAsChildren(Window window, StatusBar worker, DhPackage dhPackage)
```

Comment:
It also seems that you don't need the new window parameter (it is now unused).   I would actually like to move in the direction of not having the Window parameter (since we should be able to derive it from the StatusBar)    Thus it is better if we remove this.